luarules/gadgets/map_lava.lua

```diff
@@ -300,7 +300,7 @@ else  -- UNSYCNED
 		SPECULAREXPONENT = lava.specularExp,  -- the specular exponent of the lava plane
 		SPECULARSTRENGTH = 1.0, -- The peak brightness of specular highlights
 		LOSDARKNESS = lava.losDarkness, -- how much to darken the out-of-los areas of the lava plane
-		SHADOWSTRENGTH = lava.shadowStrength, -- how much light a shadowed fragment can recieve
+               SHADOWSTRENGTH = lava.shadowStrength, -- how much light a shadowed fragment can receive
```

Comment:
ws is made worse in a bunch of places

Reply:
Entirely up to you, though it doesn't seem like a very efficient use of anybody's mana (especially looking at your other engine PR).